docs/installation/introduction.rst

```diff
@@ -288,25 +290,25 @@ Additional Notes
 
 Sizing
 """"""
-Defining an appropriate sizing for the machines in a MetalK8s cluster strongly
-depends on the selected architecture and the expected future variations to
-this architecture. Refer to the documentation of the applications planned to
-run in the deployed cluster before completing the sizing, as their needs will
+
+Sizing the machines in a MetalK8s cluster depends on the selected architecture
+and anticipated changes. Refer to the documentation of the applications planned
+to run in the deployed cluster before completing the sizing, as their needs will
 compete with the cluster's.
 
 Each :ref:`role<node-roles>`, describing a group of services, requires a
-certain amount of resources for it to run properly. If multiple roles are used
+certain amount of resources to run properly. If multiple roles are used
 on a single Node, these requirements add up.
 
 +----------------+-----------------------+----------+--------+---------------------------------+-----------------------------+
-|      Role      |       Services        |   CPU    |  RAM   |        Required storage         |     Recommended storage     |
+|      Role      |       Services        |   CPU    |  RAM   |        Required Storage         |     Recommended Storage     |
 +================+=======================+==========+========+=================================+=============================+
 | bootstrap      | Package repositories, | 1 core   | 2 GB   | Sufficient space for the        |                             |
 |                | container registries, |          |        | product ISO archives            |                             |
 |                | Salt master           |          |        |                                 |                             |
 +----------------+-----------------------+----------+--------+---------------------------------+-----------------------------+
-| etcd           | ``etcd`` database     | 0.5 core | 1 GB   | 1 GB for                        |                             |
-|                | for K8s API           |          |        | ``/var/lib/etcd``               |                             |
+| etcd           | etcd database for the | 0.5 core | 1 GB   | 1 GB for                        |                             |
+|                | K8s API               |          |        | /var/lib/etcd                   |                             |
```

Comment:
Definitely this `/var/lib/etcd` is a path, so, monospace.

Reply:
The rule you are following is bad. When I get my hands on it, I will change it. The only reason to set a path as monospace is to indicate that it's something the reader sees or is expected to enter in a command line. For all other mentions, set it roman. Because paths are set with slashes in them, they cannot be mistaken for ordinary language. They are self-evidently paths, and do not need type effects to be easily identified as such by the reader.

Reply:
https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-filenames-directories-and-paths

That's one topic I'd like us to reconsider.

Reply:
This is not the only editorial decision by Google that gives me heartburn.

Reply:
OK, let's find some time to gather all the rules TechPubs want us to use, write it somewhere, so we don't need to debate it again :)